src/bin/agent/start.ts

```diff
@@ -14,6 +14,7 @@ const start = binUtils.createCommand('start', {
   format: true,
   passwordFile: true,
 });
+start.option('-fr, --fresh', 'Starts a fresh agent');
```

Comment:
There is probably a better way to specify the fresh option but considering the CLI commands are changing soon I've left that up to future changes